test/studies/isx/.gitignore

```diff
@@ -0,0 +1 @@
+*.good
```

Comment:
It allows the blanket ignoring of a.out, for instance.  I think there's some other common test files that are ignored en masse, and it keeps things in a centralized location.  I don't think it's a big deal, though.